src/overlays/actors/ovl_En_Heishi2/z_en_heishi2.h

```diff
@@ -3,12 +3,54 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include "overlays/actors/ovl_Bg_Gate_Shutter/z_bg_gate_shutter.h"
+#include "overlays/actors/ovl_En_Bom/z_en_bom.h"
```

Comment:
I don't think bg_gate_shutter and en_bom are needed in this header so I would only include them in the C file.